Doc/c-api/long.rst

```diff
@@ -653,3 +653,156 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
 
    .. versionadded:: 3.12
 
+
+Export API
+^^^^^^^^^^
+
+.. versionadded:: 3.14
+
+.. c:struct:: PyLongLayout
+
+   Layout of an array of "digits" ("limbs" in the GMP terminology), used to
+   represent absolute value for arbitrary precision integers.
+
+   Use :c:func:`PyLong_GetNativeLayout` to get the native layout of Python
+   :class:`int` objects, used internally for integers with "big enough"
+   absolute value.
+
+   See also :data:`sys.int_info` which exposes similar information to Python.
+
+   .. c:member:: uint8_t bits_per_digit
+
+      Bits per digit.
```

Comment:
Could you add an example of how to interpret this? Something like: "Bits per digit. For example, a 15 bit digit means that bits 0-14 contain meaningful information."

Similar for `digit_size`: "Digit size in bytes. For example, a 15 bit digit will require at least 2 bytes."

You might have better information to communicate here, my aim is to save people (i.e. me) from having to go read the PEP or the source code to figure out how to use these correctly.

Reply:
Done, I added your suggestion.